docs/index.rst

```diff
@@ -9,8 +9,16 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
 
 Overview
 --------
-.. todo::
-   Here will be the tutorial overview.
+
+The aim of this tutorial is to demonstrate how to develop a web application using the Django framework backed by Neo4j, connecting them with the Neomodel driver. Neomodel is an Object Graph Mapper (OGM) for the Neo4j graph database.
+
+The tutorial covers several topics, some of the topics are: how to configure Neomodel within a Django program, create models, query to the database, create an API. It provides best-practice guidance on implementing Neomodel.
+
+The project is a web application to search for information on a Paradise Paper Graph Database. The data from the Database includes companies and people in more than 200 countries that are part of the Paradise Papers, Panama Papers, Bahama Leaks or the Offshore Leaks investigations.
+
+Each of these data records is displayed as a `Node` within the database. If you are not familiar with the definition of a node, it is a graphic data record. There is also a term that we are going to use called `relationship` which is what connects the nodes with each others. An example of these `relationships` could be ``REGISTERED_ADDRESS`` , ``OFFICER_OF``. What’s more, each node will have `properties` that are called data values.
+
```

Comment:
A question here
Is `the nodes with each others` or `the nodes with each other`?
Sorry just want to know 😅

Reply:
each other :o you are right! good catch